content-src/components/DiscoveryStreamComponents/DSLinkMenu/DSLinkMenu.jsx

```diff
@@ -49,8 +49,8 @@ export class _DSLinkMenu extends React.PureComponent {
     return (<div>
       <button ref={this.contextMenuButtonRef}
               className="context-menu-button icon"
-              title={this.props.intl.formatMessage({id: "context_menu_button_sr"}, {title})}
-              aria-label={this.props.intl.formatMessage({id: "context_menu_button_sr"}, {title})}
+              title={this.props.intl.formatMessage({id: "context_menu_title"}, {title})}
+              aria-label={this.props.intl.formatMessage({id: "context_menu_title"}, {title})}
```

Comment:
For aria-label detailed string with title makes sense and it should be good with id as  'context_menu_button_sr'.

Looks like we need to change back to context_menu_title in Card.jsx, CollapsibleSection.jsx and TopSite.jsx. Thanks!

Reply:
Got it!

Reply:
Ah hold on - do we want aria-label to have `context_menu_button_sr` and the titles to be `context_menu_title`? I figured we'd want title and aria-label to be the same since screen readers will usually pick one or the other, so that way everyone gets the same info

Reply:
yes, my take we should keep titles to be context_menu_title and aria-label to have context_menu_button_sr , this way we will be providing detailed information where screen reader picks aria-label

Reply:
okay!